lte/gateway/python/integ_tests/federated_tests/README.md

```diff
@@ -6,87 +6,93 @@ hide_title: true
 # Federated Integration Tests (EXPERIMENTAL)
 
 The objective of Federated Integration Tests is to provide a test platform
-where to run Magma with all its components.
+to run Magma with all its components.
 
-As the diagram indicates, this test spins up AGW, FEG and Orc8r
-and uses S1aP and trf server to run tests.
+As the diagram indicates, this test spins up **AGW**, **FeG**, and **Orc8r**
+and uses `S1aP` and `magma_trfserver` as an eNb and SGi emulator to run tests.
 
 ```mermaid
 graph LR;
-    S1AP --> AGW --> orc8r --> FEG --> 3ggp_CORE;
-    3ggp_CORE --> FEG --> orc8r --> AGW --> S1AP;
-    AGW --> trf_server;
-    trf_server --> AGW;
+    A[[S1AP]] --- AGW ---|control| orc8r --- FEG --- HSS[(HSS)];
+    AGW ---|data| Z[[magma_trfserver]];
 ```
 
-The processes will run on the following machines:
-- AGW will run on **magma** vm.
-- FEG and Orc8r will run on our **local docker**.
-- Traffic server will run on **magma_trf** vm
-- Test will be started from **magma_test** vm
+The services will run either on Vagrant VM or on Docker:
+
+| Services          |  Vagrant VM       |  Docker   |
+|-------------------|:-----------------:|:----------:|
+| AGW               | magma             |           |
+| FeG               |                   | &check;   |
+| Orc8r             |                   | &check;   |
+| Traffic server    | magma_trfserver   |           |
+| S1AP tester       | magma_test        |           |
+
+
+## Build environment
+
+Next steps show you how to build the test environment automatically. If you
+want to build this step by step go to [Manual build](#Manual build)
```

Comment:
Does `#Manual build` work correctly? I think it might have to be `#manual_build` but not sure